src/Functions.php

```diff
@@ -20,6 +21,16 @@ function renderUsing(OutputInterface|null $renderer): void
     }
 }
 
+if (! function_exists('Termwind\hideOutput')) {
+    /**
+     * Hides the output for the CLI.
+     */
+    function hideOutput(): void
+    {
+        renderUsing(new BufferedOutput());
```

Comment:
You could consider using `NullOutput` instead actually 🤔 
https://github.com/symfony/console/blob/6.0/Output/NullOutput.php

Reply:
Even better! Thanks for that